impl/src/main/java/com/okta/sdk/impl/http/support/Version.java

```diff
@@ -28,39 +28,39 @@
 public class Version {
 
     private static final String CLIENT_VERSION = lookupClientVersion();
+    private static final String VERSION_FILE = "/com/okta/sdk/version.properties";
 
     public static String getClientVersion() {
         return CLIENT_VERSION;
     }
 
     private static String lookupClientVersion() {
```

Comment:
Shouldn't this include `throws RuntimeException`?

Reply:
Nope, `RuntimeExceptions` are not declared they are not a checked exception